tests/spec/cmdline_spec.rb

```diff
@@ -99,16 +99,40 @@ def self.pr_number
       expect(result).to be true
     end
   end
+end
+
+# secondary --changed_since
+describe 'cmdline changed-since' do
+  before(:each) do
+    @gitarrorepo = GIT_REPO
+    @rgit = GitRemoteOperations.new(@gitarrorepo)
+    @pr = @rgit.first_pr_open
+    @test = GitarroTestingCmdLine.new(@gitarrorepo)
+    # commit status
+    @comm_st = @rgit.commit_status(@pr)
+  end
+
+  describe '.changed_since_60' do
+    it "gitarro should see PR ##{PR_NUMBER} changed in the last 60 seconds" do
+      context = 'changed-since-60'
+      pr = @rgit.pr_by_number(PR_NUMBER)
+      comment = @rgit.create_comment(pr, "gitarro rerun #{context} !!!")
+      result, output = @test.changed_since(@comm_st, 60, context, context)
```

Comment:
in the method ```changed_since``` you are passing 2 times the same variable `context`.

You can remove the last argument

Reply:
First time for the context, second one for the description. They are using the same value, and that is why it's being used twice (instead of creating the variable twice with different names).

Otherwise I think I had the problem rubocop complaining about "method has too many lines".

In in fact, for the tests, I am not sure we need different values for context and description. Maybe we should leave it now as it is, and then refactor this and other functions.

Reply:
Of course I can change my method ```changed_since```, but we should then change the others, so they are similar.

Reply:
yop